### DISCUSSION TITLE
Do material (.mat) files support "include" files?

### DISCUSSION THREAD

**Discussion by BStringhamVRSK** [google/filament#5444] [Q&A] [answered]
I would like to create some include files with commonly-used functions.  Do material .mat files support "include" files?  If so, what is the syntax?

**Comment by romainguy** [comment, marked as answer]
You can include using #include. See example here: https://github.com/google/filament/blob/main/filament/src/materials/separableGaussianBlur1.mat#L48

**Reply by BStringhamVRSK** [reply]
Thanks!
